src/component/vocabulary/CreateVocabulary.tsx

```diff
@@ -100,12 +106,22 @@ export class CreateVocabulary extends AbstractCreateAsset<CreateVocabularyProps,
         return this.state.label.trim().length > 0;
     }
 
+    private onFileAdded() {
+        ;
+    }
+
+    private onCreateFile(termitFile: Resource, file: File): Promise<void> {
+        return Promise.resolve().then(() => {
```

Comment:
Again, what's the point of Promise.resolve() here?

Reply:
Not sure how to avoid either creating "new Promise<void>" or doing this.

Reply:
I think a much clearer way is to use return `new Promise(() => {// the executed code});`